change/@fluentui-react-components-27ab0583-ddc1-41ff-abe2-0ab8fe34d69b.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

```suggestion
  "type": "patch",
```

As it's an unstable feature, let's use "patch" instead